rasa/telemetry.py

```diff
@@ -370,40 +395,29 @@ async def _send_event(
 
     headers = segment_request_header(write_key)
 
-    async with aiohttp.ClientSession() as session:
-        async with session.post(
-            SEGMENT_ENDPOINT, headers=headers, json=payload,
-        ) as resp:
-            # handle different failure cases
-            if resp.status != 200:
-                response_text = await resp.text()
-                logger.debug(
-                    f"Segment telemetry request returned a {resp.status} response. "
-                    f"Body: {response_text}"
-                )
-            else:
-                data = await resp.json()
-                if not data.get("success"):
-                    logger.debug(
-                        f"Segment telemetry request returned a failure. "
-                        f"Response: {data}"
-                    )
-
-
-def _project_hash() -> Text:
-    """Create a hash for the project in the current working directory.
+    resp = requests.post(SEGMENT_ENDPOINT, headers=headers, json=payload)
+    # handle different failure cases
+    if resp.status_code != 200:
+        logger.debug(
+            f"Segment telemetry request returned a {resp.status_code} response. "
+            f"Body: {resp.text}"
+        )
+    else:
+        data = resp.json()
+        if not data.get("success"):
+            logger.debug(
+                f"Segment telemetry request returned a failure. " f"Response: {data}"
```

Comment:
`black` is playing pranks
```suggestion
                f"Segment telemetry request returned a failure. Response: {data}"
```